apps/api/src/index.ts

```diff
@@ -275,26 +275,28 @@ const mainServer = app.listen(PORT, "0.0.0.0", () => {
 // Start dedicated metrics server on separate port
 const metricsServer = startMetricsServer();
 
+let shuttingDown = false;
```

Comment:
I'm not sure how other systems work, but  when you hit control+c on Ubuntu the sigint listener is call twice, so I needed to add a gaurd.